simhash/__init__.py

```diff
@@ -7,10 +7,10 @@ def shingle(tokens, window=4):
     if window <= 0:
         raise ValueError('Window size must be positive')
     its = []
-    for number in xrange(window):
+    for number in range(window):
```

Comment:
I think it would be nice to use either `future` or `six` to get equivalent behavior on Python 2 and 3 regarding `xrange` vs `range`.